readme.md

```diff
@@ -436,6 +436,7 @@
 - [Educational Games](https://github.com/yrgo/awesome-eg#readme) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning#readme)
 - [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.
+- [Product Management](https://github.com/dend/awesome-product-management) - Resources for Product Managers: tools, articles, books, podcasts and more.
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
@sindresorhus - addressed this, thank you for the feedback!